programs/bpf/rust/clock/Cargo.toml

```diff
@@ -2,22 +2,21 @@
 # Note: This crate must be built using build.sh
 
 [package]
-name = "solana-bpf-rust-tick-height"
+name = "solana-bpf-rust-clock"
```

Comment:
Can I tempt you with `programs/bpf/rust/128bit/Cargo.toml` -> `programs/bpf-rust-128bit/Cargo.toml`

Reply:
No sweat, also means `programs/bpf-c/...`.  Mind if I do that in a different PR?